Azure-Samples/communication-services-web-calling-hero
# update npm dependencies for security flags

## Purpose
Updating npm dependencies that had previously enabled potential security flaws.

## Does this introduce a breaking change?
```
[ ] Yes
[x ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?
[ x] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
I updated the package.json and did an npm install and it looks like we installed the correct version.

* Test the code
Ran the sample to see that it still worked.

## What to Check
Verify that the following are valid
Verified that version of some of the updated dependencies matched what we are looking for.
